iOS Client/DevotionsController/MBDevotionsViewController.swift

```diff
@@ -64,5 +72,17 @@ class MBDevotionsViewController: UIViewController, StoreSubscriber, UITableViewD
     func numberOfSections(in tableView: UITableView) -> Int {
         return 1
     }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+       let devotion = devotions[indexPath.row]
+        MBStore.sharedStore.dispatch(SelectedDevotion(devotion: devotion))
+        do {
+            try store.saveDevotions(devotions: devotions)
+        } catch {
+            //There was an error saving devotion as read so reverse
+            print("Error marking devotion as read")
+            devotions[indexPath.row].read = false
+            MBStore.sharedStore.dispatch(LoadedDevotions(devotions: .loaded(data: devotions)))
```

Comment:
Works for now; in a multi-threaded setting we'd want to be careful about clobbering all the devotions in the state with this view controller's local copy, b/c something could change in between the dispatch selected devotion action and this . . . I guess the solution would be an additive only approach where we dispatch a new action called UnselectDevotion or something